### ISSUE TITLE
Update with Transparent UTF-8 support

### ISSUE THREAD

**Issue by Amaroq-Clearwater** [Leandros/WindowsHModular#7]
There's a mostly drop-in windows.h replacement (made by another user) which adds transparent UTF-8 support to ANSI Win32 programs, but as awesome as it is, it currently conflicts with WindowsHModular. Here is the repository in question, for review:
- https://github.com/thpatch/win32_utf8

There is also a related repository, which adds dynamic layout functionality for Win32 dialogues, but it is dependent on Win32_utf8:
- https://github.com/thpatch/win32_smartdlg

**Comment by Amaroq-Clearwater** [comment]
By the way, Microsoft has recently taken notice of both WindowsHModular and Win32_utf8, and they are both currently being reviewed for implementation in WinFile:
- https://github.com/microsoft/winfile

**Comment by Leandros** [comment]
Yes, what am I supposed to do about it? win23_utf8 does something completely different. Of course it's going to conflict.

**Comment by Amaroq-Clearwater** [comment]
What I meant was to... Like, maybe... Combine it? Add transparent UTF-8 support to the modular thing here?

I guess it's a stupid idea.

**Comment by Leandros** [comment]
It's not a goal of this library. However, the idea isn't particularly bad. It would have to be added to win32_utf8, though.
